crates/holochain_types/src/p2p_config_overrides.rs

```diff
@@ -0,0 +1,61 @@
+//!  P2P configuration overrides module.
+//!
+//! This module defines structures and functions for overriding
+//! P2P configuration settings in Holochain applications.
+
+/// Overrides for P2P configuration settings.
+///
+/// This struct holds optional override values for P2P configurations
+/// such as bootstrap URLs and signal server URLs.
+#[derive(Debug, Default, Clone, PartialEq, Eq)]
+pub struct P2pConfigOverrides {
```

Comment:
Nitpick but i find the naming "P2P configurations" confusing. Aren't all configurations related to p2p? Could it just be AppConfigOverrides or ConfigOverrides?

Feel free to ignore this if you don't agree.

Reply:
I have no idea, I knew that name sucks but I really couldn't come up with anything meaningful. I open a discussion about that, because Idk.